test/com.gu.contentapi.sanity/Config.scala

```diff
@@ -6,21 +6,21 @@ object Config {
   val conf = ConfigFactory.load()
   val sanityConfig = conf.getConfig("content-api-sanity-tests")
   conf.checkValid(ConfigFactory.defaultReference(), "content-api-sanity-tests")
-  val sanityCodeConfig = conf.getConfig("content-api-sanity-code")
-  conf.checkValid(ConfigFactory.defaultReference(), "content-api-sanity-code")
 
```

Comment:
I think this file can be safely deleted, because it is just shadowing the `Config` class (which is exactly the same as this one) under `/app`. 

All of the classes under `/app` will be on the test classpath, so the tests will find the `Config` class.